index.html

```diff
@@ -1,12 +1,732 @@
+            <div class="featured-posts__links">
+              <a class="featured-posts__links-item" href="#">health</a>
+            </div>
+            <h3 class="featured-posts__title">5 Tips To Supercharge Your Motivation</h3>
+            <div class="featured-posts__time">
+              <p class="featured-posts__date">June 14, 2015</p>
+              <div class="featured-posts__comment">
+                <a class="featured-posts__comment-link" href="#"><i class="fas fa-comment"></i> <span>24</span> </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="featured-posts__content">
+        <div class="featured-posts__wrapper">
+          <img src="./images/slide3.png" alt="health" class="featured-posts__image">
+          <div class="featured-posts__detail">
+            <div class="featured-posts__links">
+              <a class="featured-posts__links-item" href="#">Beauty</a>
+            </div>
+            <h3 class="featured-posts__title">Keep Makeup Looking Fresh Take A Powder</h3>
+            <div class="featured-posts__time">
+              <p class="featured-posts__date">June 14, 2015</p>
+              <div class="featured-posts__comment">
+                <a class="featured-posts__comment-link" href="#"><i class="fas fa-comment"></i> <span>24</span> </a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="slide-controls"></div>
+  </section>
+  <!-- end-featured-posts -->
+  <!-- page-body -->
+  <section class="page-body">
+    <!-- main-content -->
+    <div class="main-content">
+      <article class="post">
+        <div class="post__thumbnail">
+          <a href="#"><img src="./images/Art-image1.png" alt="health" class="post__image"></a>
+        </div>
+        <div class="post__description">
+          <div class="post__header">
+            <div class="post__links">
+              <a class="post__links-item" href="#">ART</a>
+              <a class="post__links-item" href="#">HEALTH</a>
+            </div>
+            <h3 class="post__title">Barbecue Party Tips For A Truly Amazing Event</h3>
+            <div class="post__time">
+              <p class="post__date">June 14, 2015</p>
+              <a href="#" class="post__author">by Author Doe</a>
+            </div>
+          </div>
+          <div class="post__content">
+            <p class="post__text">Lorem Ipsum has been the industry's standard dummy text ever since the 1500s when an unknown printer took ...</p>
+            <a href="#" class="post__more">...</a>
+          </div>
+          <div class="post__footer">
+            <div class="post__icon">
+              <div class="post__icon-left">
+                <div class="post__comment">
+                  <a href="#" class="post__comment-link"><i class="far fa-comment"></i> 24</a>
+                </div>
+              </div>
+              <div class="post__icon-right">
```

Comment:
Đặt tên lại class cho toàn bộ phần social này. Tại sao chỉ đặt là link chung chung? Link cái gì?